regression/test.pl

```diff
@@ -304,10 +304,15 @@ ($$$$)
 $main::VERSION = 0.1;
 $Getopt::Std::STANDARD_HELP_VERSION = 1;
 our ($opt_c, $opt_i, $opt_j, $opt_n, $opt_p, $opt_h, $opt_C, $opt_T, $opt_F, $opt_K, %defines, @include_tags, @exclude_tags); # the variables for getopt
-$opt_j = 0;
 GetOptions("D=s" => \%defines, "X=s" => \@exclude_tags, "I=s" => \@include_tags);
 getopts('c:i:j:nphCTFK') or &main::HELP_MESSAGE(\*STDOUT, "", $main::VERSION, "");
 $opt_c or &main::HELP_MESSAGE(\*STDOUT, "", $main::VERSION, "");
+$opt_j = $opt_j || $ENV{'TESTPL_JOBS'} || 0;
+if($opt_j && !$has_thread_pool) {
+  print "thread.pl: Jobs set but thread pool module not found,\n"
+   . "install with 'cpanm install Thread::Pool::Simple'\n"
```

Comment:
s/cpanm/cpan/

Reply:
No, I meant [cpanm](https://github.com/miyagawa/cpanminus) - see also [here](http://www.cpan.org/modules/INSTALL.html).

Reply:
That being said, `cpan` would work too and people not familiar with Perl would be more likely to have it, so I'm changing it anyway